src/main/java/school/lemon/changerequest/java/introduction/hw1/Task6.java

```diff
@@ -2,24 +2,32 @@
 
 public class Task6 {
 
-   public static int fact(int n) {
+    public static int fact(int n) {
         int result;
 
-        if(n==1) return 1;
-        result = fact(n-1) * n;
+        if (n == 1) return 1;
+        result = fact(n - 1) * n;
         return result;
     }
 
 
     public static double calculateS(double x) {
+        double sum = 0;
+
+        for (int i = 1; i < 5; i++) {
+
+
+            sum += Math.pow(x, i) / fact(i);
+        }
+
+        return sum;
```

Comment:
Check your implementation one more time - you've got failed tests. You have missed some terms in expression.